hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -661,6 +776,16 @@ set_false_path -hold -from [get_clocks SPI_TPM_IN_CLK] \
 # Remove scan paths for timing analysis
 set_clock_sense -stop_propagation -clock SPI_TPM_CLK [get_pins -leaf -filter "@pin_direction == in" -of_objects [get_nets -segments -of_objects [get_pins u_ast/u_scan_clk/in_i*]]]
 
+# Should be approved
+set_false_path -hold -from [get_clocks SPI_TPM_IN_CLK] \
+    -to [get_clocks SPI_TPM_CLK] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+        [get_cells -filter "@is_sequential" top_earlgrey/u_spi_device/u_cmdparse/*]]
+
+set_false_path -hold -from [get_clocks SPI_TPM_CLK] \
+    -to [get_clocks SPI_TPM_CLK] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+        [get_cells -filter "@is_sequential" top_earlgrey/u_spi_device/u_passthrough/cmd_info_reg_read_pipeline_mode*]]
```

Comment:
@meisnere @OTshimeon , I've asked for clarification on the status of this in the previous version of the PR. Who needs to approve this? Based on what assumptions / findings / discussions has this been added? Please comment.

Reply:
I've now removed this comment as @hcallahan-lowrisc carefully reviewed all SPI related constraints (see https://github.com/lowRISC/opentitan/pull/24221#pullrequestreview-2288609680).